Best Practices Docs Update Branch (#13887)
microsoft/fluentui: packages/office-ui-fabric-react/src/components/Tooltip/docs/TooltipBestPractices.md

### Content

- Don’t use a tooltip to restate a button name that’s already shown in the UI.

> - When a control or UI element is unlabeled, use a simple, descriptive noun phrase. For example: “Highlighting pen.” Only capitalize the first word (unless a subsequent word is a proper noun), and don’t use a period.

Reviewer: ```suggestion
- When a control or UI element is unlabeled, use a simple, descriptive noun phrase. For example: “Highlighting pen.”. Only capitalize the first word (unless a subsequent word is a proper noun), and don't use a period.
```

Reviewer 2: Fixed.

> - For a disabled control that could use an explanation, provide a brief description of the state in which the control will be enabled. For example: “This feature is available for line charts.”

Reviewer: ```suggestion
- For a disabled control that could use an explanation, provide a brief description of the state in which the control will be enabled. For example: "This feature is available for line charts".

```

Reviewer 2: In this case, there is guidance for using a period if the tooltip is a complete sentence. So the punctuation is part of the example and the quotation mark is correct.

- Only use periods for complete sentences.

For a UI label that needs some explanation:

- Briefly describe what you can do with the UI element.

> - Use the imperative verb form. For example, "Find text in this file." (not "Finds text in this file").

Reviewer: ```suggestion
- Use the imperative verb form. For example, "Find text in this file" (not "Finds text in this file").
```
